src/SWP/Component/TemplatesSystem/README.md

```diff
@@ -0,0 +1,16 @@
+Templates System [![Build Status](https://travis-ci.org/SuperdeskWebPublisher/templates-system.svg?branch=master)](https://travis-ci.org/SuperdeskWebPublisher/templates-system) [![Code Climate](https://codeclimate.com/github/SuperdeskWebPublisher/templates-system/badges/gpa.svg)](https://codeclimate.com/github/SuperdeskWebPublisher/templates-system) [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/SuperdeskWebPublisher/templates-system/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/SuperdeskWebPublisher/templates-system/?branch=master)
```

Comment:
pls use the README structure from other components so we use the same everywhere: https://github.com/SuperdeskWebPublisher/multi-tenancy/blob/master/README.md